packages/mds-db/tests/mds-db.spec.ts

```diff
@@ -396,6 +399,18 @@ if (pg_info.database) {
         assert.deepEqual(unpublishedResult, false)
       })
 
+      it('.readGeographies understands all its parameters', async () => {
+        const publishedResult = await MDSDBPostgres.readGeographies({ get_published: true })
+        assert.deepEqual(publishedResult.length, 1)
+        assert.deepEqual(!!publishedResult[0].publish_date, true)
```

Comment:
Would this suffice?
```
assert.deepEqual(!publishedResult[0].publish_date, false)
```

Reply:
I like the comparison to `true` because it seems to me to better indicate that the `publish_date` is there.

Reply:
That seems reasonable

Reply:
yeah I like `!!` better than `Boolean()` but probably the latter is more "official"